src/main/resources/application.yml

```diff
@@ -49,7 +49,9 @@ nakadi:
       rotationMs: 86400000 # 1 day
   stream:
     timeoutMs: 31536000000 # 1 year :-P
-    default.commitTimeout: 60 # 1 minute
+    default:
```

Comment:
Well, we already use this approach for other default values. But I agree that having all configurations in a single file would be ideal. If I'm doing it here, maybe we should consider no longer use `defaultValue` feature from Spring.